__stories__/legal-pages.tsx

```diff
@@ -25,7 +25,7 @@ import {
   UnrevisedPage as UnrevisedPageOriginal,
   RevisedPage as RevisedPageOriginal,
   RevisionsOverview as RevisionsOverviewOriginal,
-} from '../src/static-pages'
```

Comment:
Just a quick question: Why was static-pages renamed to legal-pages?

Reply:
In the beginning the module was created for any static page in mind (like also `/spenden`). However it is clear now that other static pages next to legal pages will be handled by the frontend. Therefore they are only used for legal pages and thus the new name is more suitable

Reply:
Thanks a lot for the clarification!